tests/modules/test_detectionlists.py

```diff
@@ -123,6 +163,16 @@ def test_remove_user_cloud_alias_calls_user_client_with_expected_values(
             TEST_USER_ID, "oldalias"
         )
 
+    def test_remove_user_cloud_alias_returns_response(
+        self, mock_microservice_client_factory, mock_detection_list_user_client
+    ):
+        mock_microservice_client_factory.get_detection_list_user_client.return_value = (
+            mock_detection_list_user_client
+        )
+        module = DetectionListsModule(mock_microservice_client_factory)
+        response = module.remove_user_cloud_alias(TEST_USER_ID, "oldalias")
+        assert response
```

Comment:
adopting this approach for testing each call we make in general will probably be a good idea going forward.

Reply:
If you want, I add them all now! But that would be a lot

Reply:
no no, wasn't suggesting that, just putting thoughts out there